youtube_dl/extractor/blick.py

```diff
@@ -0,0 +1,138 @@
+    _VALID_URL = r'(?:https?://)?(?:www\.)?blick\.ch/.*-id(?P<id>\d+).*\.html'
+
+    _TESTS = [{
+        'url': 'http://www.blick.ch/sport/uli-forte-vor-dem-abstiegs-showdown-ich-gehe-davon-aus-dass-der-fussball-gott-fcz-fan-ist-id5070813.html',
+        'info_dict': {
+            'id': '5070813',
+            'ext': 'mp4',
+            'title': 'uli-forte-vor-dem-abstiegs-showdown-ich-gehe-davon-aus-dass-der-fussball-gott-fcz-fan-ist',
+            'thumbnail': 'http://blick.simplex.tv/content/51/52/70062/simvid_1.jpg',
+            'description': 'Am Mittwochabend entscheidet sich, ob der FCZ oder der FC Lugano aus der Super League absteigt. Uli Forte schwört dabei auf den Fussball-Gott und zündet in der Kirche eine Kerze an.'
+        }
+    }, {
+        'url': 'http://www.blick.ch/sport/tennis/nominiert-fuer-musik-preis-in-schweden-so-toll-singt-guenthardts-tochter-alessandra-id5066863.html',
+        'info_dict': {
+            'id': '5066863',
+            'ext': 'mp4',
+            'title': 'nominiert-fuer-musik-preis-in-schweden-so-toll-singt-guenthardts-tochter-alessandra',
+            'thumbnail': 'http://f.blick.ch/img/incoming/crop5066860/5146024130-csquare-w300-h300/Bildschirmfoto-2016-05-23-um-14.jpg',
+            'description': 'Da ist Papa Heinz mächtig stolz. Seine Tochter Alessandra Günthardt ist für einen schwedischen Musik-Preis unter den drei Nominierten. Die Abstimmung läuft noch bis 7. Juni.'
+        }
+    }, {
+        'url': 'http://www.blick.ch/sport/fussball/superleague/totomat-fehler-in-sion-fcz-buff-stinksauer-wegen-falschem-lugano-resultat-id5063421.html',
+        'info_dict': {
+            'id': '5063421',
+            'ext': 'mp4',
+            'title': 'totomat-fehler-in-sion-fcz-buff-stinksauer-wegen-falschem-lugano-resultat',
+            'thumbnail': 'http://f.blick.ch/img/incoming/crop5063475/820602933-csquare-w300-h300/Bildschirmfoto-2016-05-22-um-19.jpg',
+            'description': 'Der FC Zürich bleibt das Schlusslicht der Raiffeisen Super League. Einen dicken Hals bekommen Buff und Co. aber wegen einer falschen Resultatanzeige aus dem Ländle.',
+        }
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        webpage = self._download_webpage(url, video_id)
+
+        found_videos = []
+        regex_og = self._og_regexes('video')
+        regex_ogs = self._og_regexes('video:secure_url')
+        video_og = self._html_search_regex(regex_og, webpage, name=None, default=None, fatal=False)
+        video_ogs = self._html_search_regex(regex_ogs, webpage, name=None, default=None, fatal=False)
+        video_meta = self._html_search_meta('contentURL', webpage, fatal=False, default=None)
+        for elem in [video_og, video_ogs, video_meta]:
+            if elem:
+                found_videos.append(elem)
+
+        video_url = ''
+        for video in found_videos:
+            if re.match(r'.*detect\.mp4', video):
```

Comment:
The problem is, that _html_search_regex only gives me the first found result, which normally contains the url to a file called "detect.mp4". But this file is not a real mp4 file like the name suggests, so I cannot use this file. I need the url to a file called "index.m3u8". I could write my own regular expression to find this file, but yan12125 suggested to use the builtin function _og_regexes, which I did...

So what should I do here?